aten/src/ATen/autocast_mode.cpp

```diff
@@ -65,6 +69,14 @@ int decrement_nesting() {
   return --nesting;
 }
 
+at::ScalarType get_autocast_dtype() {
+  return target_dtype;
```

Comment:
For consistency, is `autocast_dtype` a better name than `target_dtype`?

Reply:
Sure, change to `autocast_dtype`